File: src/AxisCRM.Api/Domain/Models/Cliente.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using AxisCRM.Api.Domain.Enums;

namespace AxisCRM.Api.Domain.Models
{
    public class Cliente : EntidadeBase
    {
        public string Nome { get; set; }
        public string CpfCnpj { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string Observacao { get; set; }
        public bool Excluido { get; set; } = false;
        public TipoPessoa TipoPessoa { get; set; }
        public DateTime? DataExclusao { get; set; }
    }
}

File: App/src/models/Cliente.js
export default class Cliente {
  constructor({
    id = 0,
    nome = '',
    tipoPessoa = null,
    cpfCnpj = '',
    telefone = '',
    email = '',
    observacao = null,
    dataCadastro = null,
    excluido = false,
    dataExclusao = null
  } = {}) {
    this.id = id;
    this.nome = nome;
    this.tipoPessoa = tipoPessoa;
    this.cpfCnpj = cpfCnpj;
    this.telefone = telefone;
    this.email = email;
    this.observacao = observacao;
    this.dataCadastro = dataCadastro ? new Date(dataCadastro) : null;
    this.excluido = excluido;
    this.dataExclusao = dataExclusao ? new Date(dataExclusao) : null;
  }
}
